# This service allows the user to enter a guess as to where they expect the
# object to be in the hand in order to speed up computation.

bool use_guess
geometry_msgs/PoseStamped guess

---

int64 ret
string msg
